### PR TITLE
First code merge

### PR DESCRIPTION
**Pull request by ctayl138**
This is the merging of our code so far. It gives us basic navigation, and a skeleton with which to work.

### CODE REVIEW

File: UI.R [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+                                plotOutput("distPlot")
+                              )
+                            )
+                   ),
+                   tabPanel("Topic 3"),
+                   tabPanel("Topic 4"),
+                   navbarMenu("More",
+                              tabPanel("Topic 5"),
+                              tabPanel("Topic 6"))
+))
```
Comment (BatesJernigan) [inline UI.R]: Add a newline at the end of this file please.

**Comment by BatesJernigan** [comment]
:+1: lgtm other than that one comment.

**Comment by BatesJernigan** [comment]
Looking forward to playing around with this tonight!

**Comment by ctayl138** [comment]
Yeah, I figure we could just designate the topics, and then go one by one, plan a layout, and put in the data visualizations even if we have to use fake data just to pin down what we're gonna do and figure out how to implement them. I know I'm gonna have to tinker a bit to figure out exactly how to do it, but so far the documentation is really friendly!

**Pull request merged**